Services/Base/IService.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.AspNetCore.Mvc;
+using QSWebAPI.Models;
+
+namespace QSWebAPI.Services.Base;
+
+public interface IService<TEntity> where TEntity : class
```

Comment:
Services are considered domain logic. An interface is not really possible in that case. Sometimes CRUD is considered domain logic in a sense, but other services may not use these functionalities.

Reply:
Should I remove the interface for Service and keep the one for  Repository?

Reply:
Yes